src/mapgen.cpp

```diff
@@ -6577,19 +6577,18 @@ character_id map::place_npc( const point &p, const string_id<npc_template> &type
 }
 
 void map::apply_faction_ownership( const point &p1, const point &p2,
-                                   const faction_id id )
+                                   faction_id id )
```

Comment:
```suggestion
                                   const faction_id &id )
```
Need according changes in the header.